src/modules/assertions/require_cc0.ts

```diff
@@ -0,0 +1,30 @@
+import { context, getOctokit } from "@actions/github";
+import { setFailed } from "@actions/core";
+import {
+  GITHUB_TOKEN,
+  GithubSelf,
+  PR,
+  Review
+} from "src/domain";
+import _ from "lodash";
+
+// BOT will fail if no CC0 detected in PR body
+// Just after: Verify correct environment and request context in main.ts, if no CC0 is detected, 
+// BOT will fail and comment: "error: No CC0 waiver detected".
+
+export const  checkCC0 = () => {
```

Comment:
This is called `checkCC0` but above it is called `require_cc0`.  I think `require_cc0` is more appropriate, as this will throw on failure, but at the least the names should be consistent.

Also, if you will be doing a lot of work on the bot I recommend getting a test environment setup so you can compile and validate changes.  I'm pretty sure this is a compiler error.

Reply:
indeed ! a test environment is the proper way to do it.